### DISCUSSION TITLE
Feature: In Modflow-2005/NWT, easily load a model with packages written internally, and change them so they are written externally.

### DISCUSSION THREAD

**Discussion by tjfarnum** [modflowpy/flopy#1810] [Q&A] [answered]
Please add an option so that a Modflow 2005/NWT model that was created with all of its packages being written internally, has an easy option of them to be rewritten externally.

Something ideal would be similar to how you can change a model workspace directory.  So something like......
example: 
model = flopy.modflow.Modflow.load('example.nam', version='mfnwt')
model.change_model_ws(new_pth='newdirectory')
model.write_all_external(package_external=True)

There seems to be a couple of psuedo-workarounds, like the model.upw.hk.how='openclose', but that only works for the upw package and not for the recharge package.  It's also time consuming to have to alter each package individually.

Thanks for looking into this.

**Comment by jtwhite79** [comment, marked as answer]
Try `model.external_path = '.' ` before writing the model input files.  That should make all arrays and lists external

**Comment by tjfarnum** [comment]
That did it.  Thanks Jeremy!
